install/installer/pkg/components/server/configmap.go

```diff
@@ -90,6 +90,19 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 		return nil
 	})
 
+	var blockedRepositories []BlockedRepository
+	_ = ctx.WithExperimental(func(cfg *experimental.Config) error {
+		if cfg.WebApp != nil && cfg.WebApp.Server != nil && len(cfg.WebApp.Server.BlockedRepositories) > 0 {
+			for _, repo := range cfg.WebApp.Server.BlockedRepositories {
+				blockedRepositories = append(blockedRepositories, BlockedRepository{
```

Comment:
Should we attempt to compile the regex to ensure it's a valid regex?

Reply:
Yes, see [d70c671](https://github.com/gitpod-io/gitpod/pull/9867/commits/d70c671c48fc6d742cf63266de2cee210735bbb0)